### DISCUSSION TITLE
Introspection is not working even tho is set to true

### DISCUSSION THREAD

**Discussion by federico-moretti** [apollographql/apollo-server#6012] [Q&A] [answered]
Hello, I'm having some trouble showing the schema introspection on a server.

In the `ApolloServer` config I've set `introspection: process.env.APPLICATION_ENV !== "production"`. In the server I've checked and that variable is different from `production` and there is no `NODE_ENV` set, yet when I go to https://studio.apollographql.com/sandbox/explorer it says "We were unable to introspect your endpoint".

Is there anything that I'm missing?

Thank you.

**Comment by federico-moretti** [comment, marked as answer]
Well, I've discovered the problem! I forgot that the API was protected, so yeah. Fixed!

**Reply by XiyanHu** [reply]
Hi I had a similar issue wonder how you managed to troubleshoot/resolve this issue, it will be very helpful. thank you!

**Reply by XiyanHu** [reply]
So I was able to retrieve a more detailed error message by using Chrome's Inspect tool network tab. It helped to root cause the issue. Just for future reference.

**Reply by isaac0yen** [reply]
Sorry but what was the exact issue?

**Reply by codal-umistri** [reply]
> Well, I've discovered the problem! I forgot that the API was protected, so yeah. Fixed!

can you describe a little because I am facing the same issue!
